cobc/ChangeLog

```diff
@@ -1,4 +1,20 @@
 
+2024-02-26  Boris Eng <boris.eng@ocamlpro.com>
+
+	* codegen.c (output_strings_states, codegen_internal):
+		added structures for the data used by strings functions as local variables
+		in the C generated code
+
+	* parser.y (examine_format_variant, inspect_region):
+		function calls of strings functions have been replaced by calls to
+		multithreaded versions (using state structures instead of static variables)
+
+	* typeck.c (cb_emit_inspect, cb_build_tallying_data, cb_build_tallying_all,
+	  cb_build_tallying_leading, cb_build_tallying_trailing, cb_build_converting,
+		cb_emit_string, cb_emit_unstring, cb_build_unstring_into:
+		function calls of strings functions have been replaced by calls to
+		multithreaded versions (using state structures instead of static variables)
```

Comment:
I'm not sure to understand what you mean by "per other entries". I see that I have spaces between each items. Is it what you are talking about?

Reply:
I meant the leading tab + two spaces

```diff
-	* codegen.c (output_strings_states, codegen_internal):
-		added structures for the data used by strings functions as local variables
-		in the C generated code
+	* codegen.c (output_strings_states, codegen_internal): added structures for
+	  the data used by string functions as local variables in generated code
+	 
 
-	* parser.y (examine_format_variant, inspect_region):
-		function calls of strings functions have been replaced by calls to
-		multithreaded versions (using state structures instead of static variables)
+	* parser.y (examine_format_variant, inspect_region), typeck.c: function calls
+	  to string functions have been replaced by calls to functions that may be
-	  called from multiple threads (using state structures instead of static
+	   variables)
 
-	* typeck.c (cb_emit_inspect, cb_build_tallying_data, cb_build_tallying_all,
-	  cb_build_tallying_leading, cb_build_tallying_trailing, cb_build_converting
-		cb_emit_string, cb_emit_unstring, cb_build_unstring_into:
-		function calls of strings functions have been replaced by calls to
-		multithreaded versions (using state structures instead of static variables)
```

(you can also list all functions for typeck.c, but as these are so much I'd personally be lazy enough to say "I've changed much of that file -> not listing all places; in any case it's the same entry as parser.y)